### PR TITLE
Document resource limits

### PR DESCRIPTION
**Pull request by neoaggelos**
#### Summary

Documentation for https://github.com/TheThingsIndustries/lorawan-stack/pull/2879

#### Screenshots

image

#### Notes for Reviewers

cc @KrishnaIyer.

Marked as draft until implementation is merged.

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [X] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/configuration/the-things-stack.md
Comment (KrishnaIyer) [inline doc/content/reference/configuration/the-things-stack.md:221]: Please fix the line endings in all occurances.

**Review by KrishnaIyer — APPROVED** [review]
LGTM

**Comment by neoaggelos** [comment]
This needs code-owner approval and is good to merge.

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**
